ThirdPartyLicenses.txt

```diff
@@ -3822,3 +3823,209 @@ END OF vscode-codicons NOTICES AND INFORATION
 
 ==============================================
 END OF openssl NOTICES AND INFORATION
```

Comment:
Typo: INFORMATION

Reply:
Thanks for catching the typo @rogererens - fixing in #1944